### PR TITLE
[Feature] add NVIDIA monitor

### PR DESCRIPTION
**Pull request by zhangshenghang**
## What's changed?

- reslove #2263

image

image

## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: home/docs/help/nvidia.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+id: nvidia  
+title: NVIDIA Monitoring  
+sidebar_label: NVIDIA Monitoring  
+keywords: [Open Source Monitoring System, NVIDIA Monitoring]
+---
+
+> Collect and monitor general performance metrics of NVIDIA operating systems.
```
Comment (tomsun28) [inline home/docs/help/nvidia.md:8]: Hi, do we need to explain here the environment that needs to be installed in advance, such as nvidia-smi

Comment (zhangshenghang) [reply home/docs/help/nvidia.md:8]: modified

File: home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+id: nvidia  
+title: 监控：NVIDIA监控
```
Comment (yuluo-yx) [inline home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md:3]: Adding a few spaces makes it easier to read and organize the document.

File: home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+> 对NVIDIA操作系统的通用性能指标进行采集监控。
+> NVIDIA监控需要用到 nvidia-smi 命令，nvidia-smi 是与 NVIDIA GPU 驱动程序一起安装的。所以在监控NVIDIA时，我们需要安装 NVIDIA GPU 驱动程序。
+
+### 配置参数
+
+|  参数名称  |                        参数帮助描述                        |
+|--------|------------------------------------------------------|
+| 监控Host | 被监控的对端IPV4，IPV6或域名。注意⚠️不带协议头(eg: https://, http://)。 |
```
Comment (yuluo-yx) [inline home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md:15]: ditto, such as `对端 IPV4，IPV6 或域名`

Comment (zhangshenghang) [reply home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md]: @yuluo-yx  already midified

File: home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md (hunk 2 above)
Comment (zhangshenghang) [reply home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/nvidia.md]: > Adding a few spaces makes it easier to read and organize the document.

@yuluo-yx  already midified

**Review by tomsun28 — APPROVED** [review]
👍

**Pull request merged**